### ISSUE TITLE
Support for intel Mac?

### ISSUE THREAD

**Issue by MelleNi** [netease-youdao/EmotiVoice#91]
Currently the Mac app only works on metal macs. Could we get an app for intel macs?

**Comment by syq163** [comment]
Certainly! We are actively working on developing an app for Intel Macs. We understand the demand from Intel Mac users and are committed to meeting their expectations. Please try using [this](https://github.com/netease-youdao/EmotiVoice/releases/download/v0.3/emotivoice-1.0.0-x86.dmg) file to see if it is compatible with your Intel Mac.

**Comment by MelleNi** [comment]
Thank you. It runs, but I am unable to change the prompt

**Comment by syq163** [comment]
Yes, prompt functionality is currently not supported in this version, but it will be enhanced in future versions.
